.github/workflows/build_test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
+        node-version: [8.x, 10.x, 12.x, 14.x, 16.x, 17.x]
```

Comment:
@amikofalvy When should we stop supporting `8` and `10`?

Reply:
IMO no, we can continue to support them until it makes our lives more difficult. As of right now that code that supports 14.x, 16.x, and 17x inherently supports 8 and 10. If that ever stops being the case we can drop them

Reply:
I agree with Kyle